src/main/java/org/commonjava/indy/service/archive/GreetingResource.java

```diff
@@ -1,4 +1,4 @@
-package org.commonjava.indy.service;
+package org.commonjava.indy.service.archive;
 
 import javax.ws.rs.GET;
 import javax.ws.rs.Path;
```

Comment:
I think this is just setup resource from quarkus, so it should not be needed anymore.

Reply:
These classes were created by @shokakucarrier , feel free to remove in the following commit if no further usage for you, here are just package refactor.

Reply:
I think it should be removed, it's generated by quarkus, I've made the change on my commit, so you can remove this change on code and wait for mine